cypress/utils/utils.ts

```diff
@@ -590,7 +590,9 @@ export function application_inventory_kebab_menu(menu, tab?): void {
 export function openManageImportsPage(): void {
     // Opens the manage import applications page
     application_inventory_kebab_menu("Manage imports");
-    cy.get("h1", { timeout: 5 * SEC }).contains("Application imports");
+    cy.wait(5 * SEC)
```

Comment:
`cy.get("h1", { timeout: 15 * SEC }).contains("Application imports");`
    doesn't work for you?

Reply:
no, it checks directly before the page finishes loading

Reply:
So the page is not rendered properly, it happens from time to time. We can apply your fix, but simultaneously we need to ask Ian to take a look and fix it on his side.

Reply:
The page does load properly, it just takes a few seconds to fully load, 
I don't think this an bug, is it ?

Reply:
Let's raise this question with Ian, but I suspect a bug here